README.md

```diff
@@ -6,7 +6,7 @@
 <table>
 <tr><th></th>              <th>Release</th> <th>Development</th> </tr>
 <tr><td>Documentation</td> <td colspan="2" align="center"><a href="https://docs.zeek.org/projects/spicy">Spicy Manual</a></td></tr>
-<tr><td>Changelog</td>     <td><a href="https://github.com/zeek/spicy/blob/v1.0.0/CHANGES">CHANGES</td> <td><a href="/CHANGES">CHANGES</a></td></tr>
+<tr><td>Release Notes</td> <td align="center"><a href="https://github.com/zeek/spicy/blob/v1.1.0/NEWS">NEWS</td> <td><a href="/CHANGES">CHANGES</a></td></tr>
```

Comment:
Let's also include the `NEWS` file literally in `doc/release-notes.rst` (probably before the `CHANGES`).

Reply:
Thinking about that, that would require to use rst for `NEWS`, maybe that makes sense given that all our docs are in rst?

Reply:
Good point, moving it over into the docs sounds good. I made it MD (1) because the commit messages use MD at least partially, and (2) so that it'd be easier to copy into the GitHub releases.  But seems the docs should be our primary place for this, and that's reST.